src/test/resources/features/convertVideo/video.feature

```diff
@@ -0,0 +1,85 @@
+Feature: Video Controller
+
+  In order to convert a file
+  As a valid Converter customer
+
+  Background: Sets authentication
+    Given I set valid authentication headers
+  @functional
```

Comment:
Please consider adding an empty line to separate background statements from scenario definition.

Reply:
I modified it according your suggestion thanks.